### ISSUE TITLE
Correlation plots and loss monitoring in fitting MLPs

### ISSUE THREAD

**Issue by laraujo-1** [duartegroup/mlp-train#174]
Hi,

I have a couple of questions regarding the training and validation of MLPs:

1. Is it possible to create correlation plots for **energies and forces** between the training and validation sets used during fitting?
2. Would it be possible to **monitor the behavior of the loss function across epochs** for both the training and validation sets? Is this already implemented?

Thank you!

Best regards,  
Lucas
